src/main/java/de/tum/in/www1/artemis/service/connectors/BitbucketService.java

```diff
@@ -50,7 +53,7 @@
     @Value("${artemis.version-control.user}")
     private String BITBUCKET_USER;
 
-    @Value("${artemis.version-control.secret}")
+    @Value("${artemis.version-control.password}")
```

Comment:
Please document somewhere that we change secret into password. This is important for all our existing Artemis servers

Reply:
I updated the PR description